lib/content-services/src/lib/aspect-list/aspect-list.component.ts

```diff
@@ -60,15 +61,10 @@ export class AspectListComponent implements OnInit, OnDestroy {
     notDisplayedAspects: string[] = [];
     hasEqualAspect: boolean = true;
 
-    private onDestroy$ = new Subject<boolean>();
+    private destroyRef = inject(DestroyRef);
```

Comment:
@dominikiwanekhyland this gives a lot of Sonar issues - could you please add readonly for each destroyRef which you introduced to resolve them? That Sonar issue is correct because you never reassign that field so should be readonly

Reply:
@dominikiwanekhyland there are still two places regarding to readonly in Sonar, could you please fix them?
<img width="1077" alt="image" src="https://github.com/user-attachments/assets/3ebb81dc-6353-42d0-b322-ee6228da91e8">